src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public static void getRandomBall() {
+        final int value = random.nextInt(100);
+        System.out.println(colorSupplier.getRandomColor() + " " + value);
```

Comment:
```suggestion
        return new Ball(colorSupplier.getRandomColor(), value);
```
you can add construcotr in Ball with 2 parameters